.github/workflows/reusable-release.yml

```diff
@@ -172,6 +107,19 @@ jobs:
         with:
           ref: ${{ matrix.branch }}
 
+      - name: install GMP
+        if: matrix.platform.DISTRO == 'Rockylinux'
+        run: |
+          set -eux
+
+          curl -O -L ${{ env.GMP_URL }}/gmp-${{ env.GMP_VERSION }}.tar.xz
+          tar xf gmp-${{ env.GMP_VERSION }}.tar.xz
+          cd gmp-${{ env.GMP_VERSION }}
+          CFLAGS=-fPIC ./configure --prefix=$HOME/.local/ --disable-shared
```

Comment:
AFAIU, static linking of the gmp library is suitable for the distribution of cabal-install since it is possible for an end user to rebuild the relevant cabal-install release (since the source code is distributed) and link against a different version of gmp. 

As I think this point is commonly misunderstood, at least linking to a description of this issue would be a good idea I think in case someone else wonders the same thing.

Reply:
I was wondering about a more practical issue: cabal doesn't use gmp itself, the dependency comes from the RTS. How safe is linking against a different version than GHC was built against?